src/lib/components/Layers/Layers.js

```diff
@@ -26,15 +40,44 @@ export class Layers extends Component {
       }
     }
 
-    layers.map((layer) => {
-      if ((!currentRegion || (layer.region && layer.region === currentRegion)) && !subLayerIds.includes(layer.id)) {
-        layerItem.push(
-          (<Layer
+    layers.forEach((layer) => {
+      if ((!currentRegion
+        || (layer.region
+          && layer.region === currentRegion))
+        && !subLayerIds.includes(layer.id)) {
+        if (layer.id) {
+          layerItem.push((<Layer
             key={layer.id}
             mapId={mapId}
             layer={layer}
-          />)
-        );
+          />))
+        } else {
+          Object.keys(layer).forEach((d, i) => {
+            layerItem = layerItem.concat([
+              (
+                <a
+                  key={`${d}-${i}-link`}
+                  className="sub-category"
+                  onClick={(e) => this.toggleSubMenu(e)}
+                >
+                  {d}
+                  <span
+                    className={`category glyphicon glyphicon-chevron-${this.state.isOpen ? 'down' : 'right'}`}
+                  />
+                </a>
+              ),
+              (this.state.isOpen ?
```

Comment:
@KipSigei - We'll need to modify the `Layers` component state to keep track of the sub-group `isOpen` state. Right now when you click one, they all open!